service/routes.py

```diff
@@ -594,43 +449,59 @@ def get(self, shopcart_id, item_id):
     @api.marshal_with(item_model)
     def put(self, shopcart_id, item_id):
         """
-        Update an Item
+        Update a Item
 
-        This endpoint will update the Item based on the posted body according to the shopcart_id and item_id specified
-        in the path.
+        This endpoint returns just a item
```

Comment:
could we reuse the original text? e.g. `This endpoint will update the Item based on the posted body according to the shopcart_id and item_id specified
        in the path.`